Makefile

```
@@ -8,7 +8,7 @@ BUILDER      = html
 JOBS         = 8
 SOURCES      =
 OUTPUT_DIR   = build
-SPHINXERRORHANDLING =
+SPHINXERRORHANDLING = -W --keep-going
```

Also nit-picky mode?

```suggestion
SPHINXERRORHANDLING = -n -W --keep-going
```

For reference:

```
  -n                nit-picky mode, warn about all missing references
  -W                turn warnings into errors
  --keep-going      with -W, keep going when getting warnings
```

---

See #3263 -- nit-picky mode is turned on via ``conf.py``!